src/ensembl/src/shared/components/error-screen/ErrorScreen.scss

```diff
@@ -36,36 +30,86 @@ $generalErrorImagesTotalWidth: calc(
   flex: 0 0 auto;
 }
 
+.generalErrorTopMessage {
+  font-weight: bold;
+}
+
 .generalErrorBody {
   flex-grow: 1;
   display: flex;
   flex-direction: column;
-  justify-content: center;
+  justify-content: flex-start;
```

Comment:
Is there any point in keeping `flex-grow: 1;` and `justify-content: flex-start;` at all anymore? The element doesn't seem to grow to the full length of the flex container, nor does it seem to need to:

![image](https://user-images.githubusercontent.com/6834224/131931890-35e69492-788a-4733-8d68-a393faf8c9cb.png)